SocialFilter/sections/specific.txt

```diff
@@ -38,6 +38,8 @@ jeotermalhaberler.com,govoritmoskva.ru,namemc.com,kevinpurcell.org,015.by,airkul
 !
 indirimdemarket.com#$##social-media-modal { display: none!important; }
 indirimdemarket.com#$#body { overflow: visible!important; }
+aiseesoft.com###share-list
+sejda.com##ul.social-list
```

Comment:
cannot find. Is tag required?

Reply:
Check my screenshot and it requires you to upload a PDF file then make some changes, apply changes, you should see socials. For Aiseesoft you gotta visit a site.